charts/internal/shoot-system-components/charts/rook-ceph/templates/operator.yaml

```diff
@@ -297,6 +297,13 @@ spec:
       - name: rook-ceph-operator
         image: {{ index .Values.images "rook-ceph" }}
         args: ["ceph", "operator"]
+        resources:
+          requests:
+            cpu: “2”
+            memory: “2Gi”
+          limits:
+            cpu: “3”
+            memory: “3Gi
```

Comment:
Something seems wrong with these quotes.
Also, those value _seem_ pretty high. Most likely way too much for a standard cluster, or?

Reply:
> Something seems wrong with these quotes.

Oops. Yeah. Fixed.

> Also, those value seem pretty high. Most likely way too much for a standard cluster, or?
 
I have not a clue. I will posit some questions.

Reply:
![Bildschirmfoto 2021-08-24 um 15 55 43](https://user-images.githubusercontent.com/738953/130629652-6955c27c-bfe1-4618-9aa8-007e72eb0e4b.png)
This is the CPU usage of our rook operator in a production cluster with 300TB. So, not much going on ;-)

Reply:
OK, dropped it by a lot